GOVERNANCE.md

```diff
@@ -162,7 +186,8 @@ To nominate a new Collaborator:
 4. Open an issue in the [nodejs/node][] repository. Provide a summary of
    the nominee's contributions (see below for an example). Mention
    @nodejs/collaborators in the issue to notify other collaborators about
-   the nomination.
+   the nomination. _Ideally_, this issue should not be opened until any private
+   discussion (questions, concerns, objections, etc) has been resolved.
```

Comment:
Is this addition necessary given the suggested comment already says that? In any case, I would recommend putting it before "open an issue", otherwise they might already have opened the issue when reading this sentence.